cmake/OpenCVModule.cmake

```diff
@@ -800,11 +800,35 @@ macro(_ocv_create_module)
     endforeach()
   endif()
 
+  if(WIN32 AND BUILD_SHARED_LIBS AND NOT OPENCV_VS_VERSIONINFO_SKIP)
+    if(DEFINED OPENCV_VS_VERSIONINFO_FILE)
+      set(_VS_VERSION_FILE "${OPENCV_VS_VERSIONINFO_FILE}")
+    elseif(DEFINED OPENCV_VS_VERSIONINFO_${the_module}_FILE)
+      set(_VS_VERSION_FILE "${OPENCV_VS_VERSIONINFO_${the_module}_FILE")
+    elseif(NOT OPENCV_VS_VERSIONINFO_SKIP_GENERATION)
+      set(_VS_VERSION_FILE "${CMAKE_CURRENT_BINARY_DIR}/vs_version.rc")
+      ocv_generate_vs_version_file("${_VS_VERSION_FILE}"
```

Comment:
how is it used later? I mean how is it embedded into DLL?

Reply:
.rc file is added as a source file (through `add_library()`)